### DISCUSSION TITLE
Put JuliaDataScience cheat sheet into docs

### DISCUSSION THREAD

**Discussion by SimonDanisch** [JuliaPlots/Makie.jl#4229] [Ideas]
Showing @lazarusA cheat sheet in the docs will be really valuable for first time users.
I imagine, we should inline this into the docs (somewhere close to the startpage), with the titles (e.g. the `scatter(x,y)`) linking to the full code example, and then having a pdf version to download.

Hosted here right now: https://juliadatascience.io/datavisMakie_attributes

![image](https://user-images.githubusercontent.com/1010467/210597867-b4638e18-a6f1-4174-bb68-6c9f20485b66.png)
![image](https://user-images.githubusercontent.com/1010467/210597895-78183204-f7ad-4645-82b2-1c8f859a178b.png)

Source:
https://github.com/JuliaDataScience/JuliaDataScience/blob/main/src/cheatSheet_cairo.jl
https://github.com/JuliaDataScience/JuliaDataScience/blob/main/src/cheatSheet_glmakie.jl

**Comment by jkrumbiegel** [comment]
We do have this page https://docs.makie.org/stable/examples/plotting_functions/

I'm partial to the titles because often we have multiple different options for input arguments (plus conversions) so it is a little more involved.

**Comment by AlexisRenchon** [comment]
Here are my thoughts: 
I think @lazarusA Cheat Sheet could fill a need not covered by the existing [plotting_functions](https://docs.makie.org/stable/examples/plotting_functions/) page, but needs some edits. 

The current examples in the plotting_functions page are neat, but they are not the most basic you can imagine, e.g., the 1st scatter example uses `range` and `.`, which may confuse a total beginner. Some examples are quite 'advanced'. 

A beginner may want something super simple, to print and put on the wall next to his computer =) Cheat Sheet also allow to have a lot of example in 1 page - no need to navigate a web menu. 

What I would add to Lazaro Cheat Sheet is 1 or 2 lines on top to give simple example of each argument (e.g., x = [1,2,3], y = [2,5,2], color = :blue, string = "hello", ...) it would of course not produce the figure shown, but it would help a beginner with extremely simple example of what these arguments can be.

**Comment by jkrumbiegel** [comment]
That's a good observation. For me, the docs should be made as accessible as possible. So if the scatter examples are too complicated, we should work to simplify them. And the overview page could also use some polish. A cheat sheet still has its place, but it shouldn't be a remedy for badly structured docs.
